google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -663,7 +663,7 @@ public String toString() {
             .toString();
       }
 
-      StorageClass getStorageClass() {
+      public StorageClass getStorageClass() {
```

Comment:
only this things would change as public rest of code will not change

Reply:
okay